nixos/modules/services/networking/nextdns.nix

```diff
@@ -12,6 +12,13 @@ in {
         default = false;
         description = "Whether to enable the NextDNS DNS/53 to DoH Proxy service.";
       };
+      package = mkOption {
+        default = pkgs.nextdns;
+        defaultText = "pkgs.nextdns";
+        example = "pkgs.nextdns";
```

Comment:
This isn't a good example - it is the default. Please provide an example that's different than the default.

Reply:
This new example makes me think what you're really looking for is overlays, not a `package` option :thinking: